guides/resource-provisioning/postgres-database.md

```diff
@@ -0,0 +1,189 @@
+# Postgres Database
+
+This guide describes how to add a Postgres database to a microservice running on the FFC Platform Kubernetes cluster.
+
+## Create Managed Identity for your microservice
+
+If not alreay configured add [Managed Identity](managed-identity.md) to your microservice
+
+## Request creation of microservice database
+
+Request Cloud Services through your usual support channel to create a database within your Azure Database for PostgreSQL following the naming convention:
+
+```
+ffc_<workstream>_<service>
+```
+
+for example `ffc_demo_claim`.
+
+*Note here the use of underscores instead of the normal hyphen convention. Postgres hyphens require escaping with double quote marks so underscores are preferred.*
+
+## Request creation of microservice database role
+
+Request Cloud Services to create a database role that is bound to the Managed Identity created for the microservice ([Azure guidence](https://docs.microsoft.com/en-us/azure/postgresql/howto-connect-with-managed-identity#creating-a-postgresql-user-for-your-managed-identity)), for example `ffc-snd-demo-claim-role`.
+
+## Create a Liquibase changelog
+
+The FFC Platfrom CI and deployment pipelines support database migrations using [Liquibase](https://www.liquibase.org/).
+
+Create a Liquibase changelog defining the structure of your database available from the root of your microservice repoository in `changelog/db.changelog.xml`.
+
+Guidence on creating a Liquibase changelog is outside of the scope of this guide, so please check current best practice with the FFC Platform Team.
```

Comment:
I think this highlights that we need standards on Liquibase migrations included in this repo.  To a certain extent I want teams to be free to structure them what works best for them but there are some prerequisites CI needs such as:

- migrations must be versioned
- version 0.0.0 must exist as the first changeset
- must be in `changelog` folder
- must have the scripts
- must have the compose file

Reply:
Agreed. Do you want me to update anything here, or are you OK with my cop-out statement?

Reply:
Leave it as is, I'll add a separate standard and add a link here retrospectively